lms/djangoapps/discussion_api/serializers.py

```diff
@@ -191,12 +191,12 @@ class ThreadSerializer(_ContentSerializer):
     closed = serializers.BooleanField(read_only=True)
     following = serializers.SerializerMethodField()
     comment_count = serializers.SerializerMethodField(read_only=True)
```

Comment:
@robrap unfortunately case 1 was missed where `comment_count` should be equal to `unread_comment_count`.
I have made an update to the PR. Kindly review.